Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Microsoft 365/Bases/PaletteMicrosoft365Base.cs

```diff
@@ -168,6 +166,46 @@ private static readonly Padding
 
         #endregion
 
+        #region Image Array
+
+        private static readonly Image[] __buttonSpecImages = new Image[]
+        {
```

Comment:
These will still need magic numbers to access (Ok, they may be enums' but what if one of these palettes introduces an image in the middle?)

Reply:
I think that a 'key & value' solution would be better.